src/component-queries/convert-data.ts

```diff
@@ -2,13 +2,36 @@ import {
     DiseaseCellLineNode,
     NormalCellLineNode,
     UnpackedDiseaseCellLine,
+    UnpackedGene,
+    GeneFrontMatter,
     UnpackedNormalCellLine,
 } from "./types";
 
+
+const extractGenes = (geneArray: { frontmatter: GeneFrontMatter }[] = []): UnpackedGene[] => {
+    if (!geneArray) return [];
+
+    return geneArray
+        // TODO: check with scientists
+        // filter out null values, e.g. UBTF, CLYBL-dCas9-KRAB 
```

Comment:
this should be removed, you can make it a todo, but you shouldn't be able to add a gene to a cell line without first adding it to the collection